README.md

```diff
@@ -3,9 +3,9 @@
 
 # Lima: Linux virtual machines (on macOS, in most cases)
 
-Lima launches Linux virtual machines with automatic file sharing, port forwarding, and [containerd](https://containerd.io).
+Lima launches Linux virtual machines with automatic file sharing, port forwarding (similar to WSL2), and [containerd](https://containerd.io).
```

Comment:
Just nit picking, but I would maybe word it like this, to make it clear that both file sharing and port forwarding are similar to WSL2. But I'm also fine with leaving it as-is.

```suggestion
Lima launches Linux virtual machines with automatic file sharing and port forwarding (similar to WSL2), and [containerd](https://containerd.io).
```

Reply:
Thanks, updated